core/src/main/scala/kafka/coordinator/GroupMetadataManager.scala

```diff
@@ -141,7 +141,8 @@ class GroupMetadataManager(val brokerId: Int,
       // if we crash or leaders move) since the new leaders will still expire the consumers with heartbeat and
       // retry removing this group.
       val groupPartition = partitionFor(group.groupId)
-      val tombstone = new Message(bytes = null, key = GroupMetadataManager.groupMetadataKey(group.groupId))
+      val tombstone = new Message(bytes = null, key = GroupMetadataManager.groupMetadataKey(group.groupId),
+        timestamp = SystemTime.milliseconds, magicValue = Message.MagicValue_V1)
```

Comment:
Regarding the message format check. Currently we do validate both broker and topic level configuration to make sure message format version is on or below inter-broker protocol version. So  is it sufficient to simply use the message format version in the config?